schema.graphql

```diff
@@ -594,16 +597,6 @@ type UserNode implements Node {
   secondaryEmail: String
 }
 
-type UserNodeConnection {
-  pageInfo: PageInfo!
-  edges: [UserNodeEdge]!
-}
-
-type UserNodeEdge {
-  node: UserNode
-  cursor: String!
-}
-
```

Comment:
We should add this back in properly in a bit. Would be really handy if we could work out how to change the UserNode for this package.